### DISCUSSION TITLE
How to use Istio without Kubernetes or Docker ?

### DISCUSSION THREAD

**Discussion by scstc** [istio/istio#53182] [Q&A]
How to use Istio without Kubernetes or Docker ?

**Comment by howardjohn** [comment]
Istio is not super well suited for running without Kubernetes *at all*. Usually people will have Kubernetes somewhere and then attach non-k8s workloads, since K8s provides the backing API. There is some ability to run entirely dettached though, https://blog.howardjohn.info/posts/local-gateway/ is an old blog about it. That uses docker https://github.com/howardjohn/local-istio-development covers some non-docker, though its primarily targeting development not production usage

**Reply by scstc** [reply]
thanks
